cora_description/urdf/macros.xacro

```diff
@@ -5,7 +5,8 @@
 
   <!-- Include sensor macros -->
   <xacro:include filename="$(find wamv_gazebo)/urdf/sensors/wamv_camera.xacro" />
-  <xacro:include filename="$(find wamv_gazebo)/urdf/sensors/wamv_gps.xacro" />
+  <!-- Modified for marina world -->
+  <xacro:include filename="$(find cora_description)/urdf/sensors/wamv_gps.xacro" />
```

Comment:
Should we change the file name to `cora_gps.xacro`?

Reply:
It's the same content apart from the GPS coordinates in it being different. If you wanted to leave it I guess that it would require that wamv_gps.xacro accept a lat/lon argument on launch? Otherwise the model coords won't match.

Reply:
Thank you, I've modified things such that the only difference now should be in `cora.xacro` where line 63 has been amended to `<xacro:wamv_gps name="gps" x="-1.0" z="4.6" lat="44.091722" lon="9.823564"/>`

Git skills are still coming up to speed, so it might look a little messy.